packages/css/src/index.ts

```diff
@@ -284,8 +296,8 @@ export const css = (args: ThemeUIStyleObject = {}) => (
     const transform: any = get(transforms, prop, get)
     const value = transform(scale, val, val)
 
-    if (multiples[prop]) {
-      const dirs = multiples[prop]
+    if (prop in multiples) {
```

Comment:
This line is a nice example of how sometimes common JS doesn't translate to strict TypeScript.

I'd guess the reasoning is: Since we don't know if `prop` is a key of `multiples`, we shouldn't just try to retrieve it without checking. Getters can have side effects!